config/default.pl

```diff
@@ -44,4 +48,33 @@ =head1 CONFIGURATION EXAMPLE
     },
   };
 
+=head2 compile
+
+The compile section defines the behaviour under which your code is run via the
+C<-c> flag.
+
+By default, we add C<lib> and C<local/lib/perl5> to C<@INC>.  If you would like to add to these paths, use something like this configuration:
+
+  my $config = {
+      compile => {
+          inc => ['foo/bar' 'my-custom-lib' ],
+          replace => 0,
+      },
```

Comment:
Looking at this now, I think this should be:
```perl
my $config = {
    compile => {
        inc => {
            libs => ['foo/bar', 'my-custom-lib'],
            replace => 0,
        },
    },
},
...
```

That would make it easier to add other config for compile later on, as `replace` is ambiguous.  What do you think?